app/src/main/java/org/fossasia/openevent/general/data/Resource.kt

```diff
@@ -13,4 +15,9 @@ class Resource {
 
     fun getString(@StringRes resId: Int, vararg args: Any?) = context.getString(resId, args)
 
+    fun isNetworkConnected(): Boolean {
```

Comment:
This should be in a separate class called Network. This is not a responsibility of Resource